lib/game_statistics.rb

```diff
@@ -18,7 +18,7 @@ def all_games
 
   def total_score
     all_games.map do |games|
-      games.home_goals + games.away_goals
+      games.home_goals.to_i + games.away_goals.to_i
```

Comment:
I see that we had to set these back to .to_i due to the removal of the numeric converter (right?)

Reply:
yep!